tracktotrip/location.py

```diff
@@ -21,7 +21,7 @@
 FS_CACHE = {}
 
 def from_cache(cache, point, threshold):
-    for entry in cache.keys():
+    for entry in list(cache.keys()):
```

Comment:
There's no need to convert a set into a list.

Reply:
Automatically done by 2to3, I don't think it makes any difference, but can be done to have no conversion.